### DISCUSSION TITLE
Slack link is not working

### DISCUSSION THREAD

**Discussion by cylin928** [pyg-team/pytorch_geometric#5556] [Q&A]
I got "This link is no longer active" when I tried to join the slack community.

**Comment by EdisonLeeeee** [comment]
Hi, does this link work? https://torchgeometricco.slack.com/

**Comment by cylin928** [comment]
I got "It looks like there isn’t an account on PyG tied to this email address."
I think I need to be invited first to be able to sign in.

**Comment by EdisonLeeeee** [comment]
See https://github.com/pyg-team/pytorch_geometric/issues/5557. @rusty1s just fixed it.
